Exception: unknown tag: ColWidth

Apologizes if I'm reporting in the wrong repository.

When following your [tutorial](https://entangled.github.io/tutorial.html), and after installing everything properly, I stumbled upon this error:
```sh
make site
pandoc --template style/template.html --css css/bootstrap.css --css css/mods.css -t html5 -s --mathjax --toc --toc-depth 1 --filter pandoc-bootstrap -f markdown+multiline_tables+simple_tables lit/index.md -o docs/index.html
Traceback (most recent call last):
  File "/home/ngirard/.local/bin/pandoc-bootstrap", line 8, in <module>
    sys.exit(main())
  File "/home/ngirard/.local/lib/python3.8/site-packages/entangled/bootstrap.py", line 160, in main
    run_filters([bootstrap_card_deck, bootstrap_fold_code], prepare=prepare, doc=doc)
  File "/home/ngirard/.local/lib/python3.8/site-packages/panflute/io.py", line 233, in run_filters
    doc = load(input_stream=input_stream)
  File "/home/ngirard/.local/lib/python3.8/site-packages/panflute/io.py", line 56, in load
    doc = json.load(input_stream, object_pairs_hook=from_json)
  File "/usr/lib/python3.8/json/__init__.py", line 293, in load
    return loads(fp.read(),
  File "/usr/lib/python3.8/json/__init__.py", line 370, in loads
    return cls(**kw).decode(s)
  File "/usr/lib/python3.8/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python3.8/json/decoder.py", line 353, in raw_decode
    obj, end = self.scan_once(s, idx)
  File "/home/ngirard/.local/lib/python3.8/site-packages/panflute/elements.py", line 1478, in from_json
    raise Exception('unknown tag: ' + tag)
Exception: unknown tag: ColWidth
Error running filter pandoc-bootstrap:
Filter returned error status 1
make: *** [Makefile:53 : docs/index.html] Erreur 83
```

Any ideas ?

Cheers

--- a/setup.py
+++ b/setup.py
@@ -17,7 +17,7 @@ setup(
     install_requires=[
           "jupyter_client>=5.3.4,<6"
         , "pampy>=0.3.0,<0.4"
-        , "panflute==1.11.2"
+        , "panflute==2.3.0"
         , "ansi2html"
     ],
     tests_require=test_deps,

